src/shared/services/form/field-type-sanitiser.ts

```diff
@@ -13,33 +13,33 @@ export class FieldTypeSanitiser {
    */
    sanitiseLists(caseFields: CaseField[], editForm: any) {
 
-    this.getDynamicListsFromCaseFields(caseFields).forEach(dynamicField => {
-      this.getListOfKeysFromEditForm(editForm).forEach((key) => {
-        this.createValueCodePairAlongWithListIfKeyExistsInForm(dynamicField, key, editForm);
-      });
+    caseFields.forEach(caseField => {
+      if (caseField.field_type.type === 'DynamicList') {
```

Comment:
Please use existing constant instead of literal string

Reply:
done